build.sh

```diff
@@ -158,9 +158,11 @@ for ARG in "$@"; do
         --rel|--release     ) flags="$flags -DCMAKE_BUILD_TYPE=Release" ; build="$root/build_rel";;
         --prof              ) flags="$flags -DCMAKE_BUILD_TYPE=RelWithDebInfo"; build="$root/build_prof";;
         --strip             ) flags="$flags -DCMAKE_BUILD_STRIP=TRUE" ;;
-        --use-system-yaml   ) flags="$flags -DUSE_SYSTEM_YAML=TRUE" ;;
         --asan              ) flags="$flags -DUSE_ASAN=TRUE" ;;
         --gtk2              ) flags="$flags -DGTK2_API=TRUE" ;;
+        --qt6               ) flags="$flags -DQT_BUILD=TRUE" ;;
+        --use-system ) flags="$flags -DUSE_SYSTEM_LIBS=ON" ;;
+        --use-bundled ) flags="$flags -DUSE_SYSTEM_LIBS=OFF" ;;
```

Comment:
Align these two like the rest of the options?
```suggestion
        --use-system        ) flags="$flags -DUSE_SYSTEM_LIBS=ON" ;;
        --use-bundled       ) flags="$flags -DUSE_SYSTEM_LIBS=OFF" ;;
```

Reply:
Take care of.